services/player/youtube-player.js

```diff
@@ -1,6 +1,8 @@
 const Player = require('./player');
 const Discord = require('discord.js');
 const fs = require('fs');
+const path = require("path");
+const crypto = require("crypto");
```

Comment:
For consistency please consider using single quotes

Reply:
I'm not so concerned with OCD nitpicky things like this.

If it pleases your senses to change to single quotes here, I'm not offended in the least.

Reply:
I know that code reviews can be discouraging and demotivating. And this is not a major thing, but staying consistent and having a pattern in the smallest things of code. Yes, even quote usage. It really helps make it tidier and predictable.

Reply:
It's neither discouraging nor demotivating -- I just don't care about unimportant things.

I'm not partial to using double-quotes here, and you can easily change them to single-quotes with a quick commit if you decide to merge these changes.

Reply:
With all due respect if I have to clean up after your commits what's the point of merging them altogether. 

Thanks for making one nonetheless.

Reply:
I think the point of merging them is that it takes your code from barely working at all to actually working. The parts of the code that actually do something are the parts that would be beneficial to integrate.